### PR TITLE
25 static contracts page

### LINKED ISSUE #25: Static contracts page
## Description
As a purchaser I want to view the available contracts so that I can see my discounts.

## Acceptance criteria
- [x] There is a page for contracts
- [x] There is a single contract visible
- [x] The contract is fetched from the backend
- [x] The backend fetches the fake contract from a mocked database

## Dependencies
- #8 
- #9

### PR DESCRIPTION
**Pull request by jonathan-lindqvist**
## Summary
Add a new contract page that fetches contracts from a faked database.

## Definition of done
- [x] All acceptance criteria are completed.
- [ ] The code has (at least) 80% branch test coverage.
- [x] All tests pass.
- [ ] The code is reviewed and accepted by (at least) two other team members.

Closes #25

### CODE REVIEW

File: tests/Application.Tests/Contracts/ContractServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using Application.Contracts;
+using Domain.Contracts;
+
+namespace Application.Tests.Contracts;
+
+public class ContractServiceTests
+{
+    [Fact]
+    public void FetchAllContracts_ReturnsALlContractsInTheDatabase()
```
Comment (JacobBredin) [inline tests/Application.Tests/Contracts/ContractServiceTests.cs:9]: Typo in ReturnsALl

**Review by JacobBredin — APPROVED** [review]
LGTM! found one typo 🖊️

**Review by martinjonsson01 — COMMENTED** [review]
Reviewing the parts I didn't write myself:

**Review by pkasinski — APPROVED** [review]
LGTM. As previously mentioned there is a typo to fix.

**Pull request merged**